prow/gerrit/adapter/adapter.go

```diff
@@ -506,6 +508,23 @@ func (c *Controller) handleInRepoConfigError(err error, instance string, change
 	return nil
 }
 
+// shouldSkipProcessingChange determines if we chan skip processing the change when there is no new commit or relavent commands in the comment messages
+func (c *Controller) shouldSkipProcessingChange(instance string, change client.ChangeInfo) bool {
+	lastUpdate, exists := c.tracker.Current()[instance][change.Project]
+	if !exists {
+		lastUpdate = time.Now()
+	}
+
+	for _, message := range currentMessages(change, lastUpdate) {
+		if pjutil.TestRe.MatchString(message.Message) || pjutil.RetestRe.MatchString(message.Message) || pjutil.TestAllRe.MatchString(message.Message) {
```

Comment:
The `/test all` regexp should be a subset of the `/test` regexp so lets remove that redundancy.

Reply:
We decided to keep the `TestAll` regex per offline discussion.